### PR TITLE
Remove old todos

### PR DESCRIPTION
**Pull request by ndricimrr**
**Description**

Changes proposed in this pull request:

- Remove old todos that are out of context or outdated

part of #3532

### CODE REVIEW

File: container/src/LuigiCompoundContainer.svelte [hunk 1]
```diff
@@ -127,7 +127,7 @@
       compound: compoundConfig,
       viewUrl: viewurl,
       webcomponent: GenericHelperFunctions.checkWebcomponentValue(webcomponent) || true
-    }; // TODO: fill with sth
```
Comment (ndricimrr) [inline container/src/LuigiCompoundContainer.svelte:130]: missing context

File: container/src/services/container.service.ts [hunk 2]
```diff
@@ -107,15 +107,14 @@ export class ContainerService {
...
-                    authData: targetCnt.authData || {},
+                    authData: targetCnt.authData || {}
                   },
                   '*'
                 );
                 break;
               case LuigiInternalMessageID.NAVIGATION_REQUEST:
                 this.dispatch(Events.NAVIGATION_REQUEST, targetCnt, event.data.params);
                 break;
-              // TODO 1: handle alerts with ids on next iteration
```
Comment (ndricimrr) [inline container/src/services/container.service.ts:118]: already handled

File: container/src/services/container.service.ts [hunk 3]
```diff
@@ -164,7 +163,6 @@ export class ContainerService {
               case LuigiInternalMessageID.GET_CURRENT_ROUTE_REQUEST:
                 this.dispatch(Events.GET_CURRENT_ROUTE_REQUEST, targetCnt, event);
                 break;
-              // TODO: discuss if actually needed as the only scenario is when microfrontend initially starts
```
Comment (ndricimrr) [inline container/src/services/container.service.ts:167]: needed as it needs to fit to existing LuigiClient

File: core/src/App.svelte [hunk 4]
```diff
@@ -746,7 +746,6 @@
         msg: 'luigi.ux.alert.hide',
         id,
         dismissKey
-        //TODO: update docu for this param
```
Comment (ndricimrr) [inline core/src/App.svelte:749]: outdated [from long time ago](https://github.com/SAP/luigi/pull/480/files#diff-c7697a12c9830e85a6f12e0bf2c589866d6d878a5f6f2c594fd4540937b09229R696)

File: container/src/services/webcomponents.service.ts [hunk 5]
```diff
@@ -364,7 +364,7 @@ export class WebComponentService {
 
     if (wc.__postProcess) {
       const url =
-        new URL(document.baseURI).origin === new URL(viewUrl, document.baseURI).origin // TODO: check if needed
```
Comment (ndricimrr) [inline container/src/services/webcomponents.service.ts:367]: out of context

**Review by walmazacn — APPROVED** [review]
LGTM 👍

**Pull request merged**